_data/events.yaml

```diff
@@ -12,5 +12,20 @@
   people:
```

Comment:
I would be eager to move that as a proper Jekyll collection with a Markdown file for each event, stored in a `_events` folder. 
If it is an OLS event, it will get its page
If the event is external, we can use `external_link` metadata to link to it.

Reply:
Actually, in my initial implementation of this page, I used a collection `_events` and added the files under that folder. Somehow, when the website was served up, the pages didn't appear at all.

I will try that again...

Reply:
Okay, it works now! 🎉 
I realise that the `permalink` was problematic. The idea was to make the path show the date and event title, but the implication is that the other pages without dates (e.g index, internal- & external-events) could not be displayed.

I hope that rambling made sense.

Reply:
We might need only a `events.md` at the root if we have the table where it shows clearly the events organized by OLS or not (and maybe have a way to filter the table). Then no need for the 3 pages without dates. What do you think?

Reply:
💭 We have three pages right now because we want to display `internal-` & `external-events` in separate pages. If this were not the case, we could have only one file (`index.md`) with the table.

Reply:
Let's start first simple with just one page listing everything. We can add other pages later

Reply:
Okay, sure. 👍

Reply:
I do have a concern. I'm not so comfortable with the current structure of the `events.yaml` file.

Reply:
I would do it the other way. Each event is a new markdown file in a `_events` folder. It can be just with metadata on it. And in the metadata, I would add one `ols_team_participation` (or something like this) with several properties in it: `type`, `title`, `who`, etc

Reply:
That makes a lot of sense!

It does mean we do not need `events.yaml`, but rather individual files with unique metadata.
Then we can loop through these files within the `_events` folder to populate the table in the `events.md` page.

Reply:
The absence of an `events.yaml` file also reduces the likelihood of automation.
By this, I mean that we cannot automatically extract data from an **events planning sheet**, and add it directly to a yaml file that will handle the process of populating the table. Events must be manually added, before they appear on the site.

Did I communicate that concern properly?

Reply:
We can still automatize the event creation. Instead of filling the yaml file, we create a new file inside the folder with the metadata on the top

Reply:
Good idea 💡
Similar to how we display the work streams...

Reply:
I did not get your last sentence

Reply:
I was referring to how you added metadata in the [three-pillars streams](https://github.com/open-life-science/open-life-science.github.io/blob/998e3aa087fb59ea973158c32b5bba30580ce0ee/open-research.md?plain=1#L9).

Reply:
You should probably more think it like the posts, but this sometimes only metadata and no Markdown content

Reply:
With also the year, month, short title in the file name

Reply:
Yes yes. I'm thinking in this line too.